src/main/kotlin/model/Delius.kt

```diff
@@ -54,7 +74,9 @@ class Delius private constructor() {
       ).apply {
         url = "https://github.com/ministryofjustice/community-api"
         uses(database, "connects to", "JDBC")
-        ec2.add(this)
+        uses(deliusApi, "Writes data to nDelius using")
+        uses(HMPPSAuth.system, "Authenticates using")
```

Comment:
Is it worth having container-to-container relationships by using `HMPPSAuth.app`?

Reply:
👍